base/bootstrap.go

```diff
@@ -78,7 +78,10 @@ func (cc *CouchbaseCluster) GetConfigBuckets() ([]string, error) {
 		return nil, errors.New("nil CouchbaseCluster")
 	}
 
-	buckets, err := cc.c.Buckets().GetAllBuckets(nil)
+	buckets, err := cc.c.Buckets().GetAllBuckets(&gocb.GetAllBucketsOptions{
+		Timeout:       time.Second * 10,
```

Comment:
Make a cluster-wide timeout instead of per-op.

Reply:
Doing this depends on the changes in #5218 - closing PR and opening Jira ticket instead.